include/faabric/transport/MessageEndpoint.h

```diff
@@ -13,10 +13,9 @@
 #define ANY_HOST "0.0.0.0"
 
 // These timeouts should be long enough to permit sending and receiving large
-// messages, note that they also determine the period on which endpoints will
-// re-poll.
-#define DEFAULT_RECV_TIMEOUT_MS 60000
-#define DEFAULT_SEND_TIMEOUT_MS 60000
```

Comment:
These were the same, and not used consistently, so I merged it into a single constant.